services/web/server/src/simcore_service_webserver/rest/handlers/basic.py

```diff
@@ -0,0 +1,74 @@
+"""This is a generated stub of handlers to be connected to the paths defined in the API
+
+"""
+# TODO: exceptions while developing ...
+# pylint: disable=unused-argument
+# pylint: disable=unused-import
+
+import logging
+import pkg_resources
+
+from aiohttp import (
+    web
+)
+from aiohttp_security import (
+    remember, forget,
+    has_permission, login_required
+)
+
+from ..config import (
+    API_URL_VERSION,
+    api_version
+)
+from .._generated_code.models.health_check import HealthCheck
+from .._generated_code.models.health_check_enveloped import HealthCheckEnveloped
+
+from ... import decorators
+
+log = logging.getLogger(__name__)
+
+async def check_health(request):
+    distb = pkg_resources.get_distribution('simcore-service-webserver')
+    # TODO: add state of services (e.g. database not connected/failed, etc)
+
+    # TODO: unify location of service info. setup.py should take if from there?
+    info = HealthCheck(
+        name = distb.project_name,
+        status = "running",
+        version = distb.version,
+        api_version = api_version())
+
+    return HealthCheckEnveloped(data=info, status=200)
```

Comment:
you do not need to call .to_dict() here?

Reply:
Really? I hope this is temporary until we extend apiset ... otherwise it is kind of half-way done. Wouldn't you agree??

Reply:
after double-checking: you only need to use to_dict() when the status is not 200... that is this issue with Jsonify

Reply:
ok